### ISSUE TITLE
welcome Peter

### ISSUE THREAD

**Issue by martindurant** [fsspec/kerchunk#166]
@peterm790 is the new GSoC student for kerchunk, working on "Efficient access to IOOS data in the cloud: interfacing WRF forecast grib2 data to zarray". Please welcome all!

You can also join the conversation in gitter: [pangeo-data/cloud-performant-netcdf4](https://gitter.im/pangeo-data/cloud-performant-netcdf4).

**Comment by peterm790** [comment]
Hi @martindurant,

Thank you very much for accepting my proposal and welcoming me to the community I look forward working with you all!

**Comment by rsignell-usgs** [comment]
@peterm790 , I scheduled a GSoC Kerchunk meet & greet for 9am ET tomorrow (1PM UTC) with @martindurant and myself, just in case you can make it.   

I didn't have your email, so please ping me at rsignell@usgs.gov and I'll add you to the invite!

**Comment by peterm790** [comment]
Hello @rsignell-usgs, yes 1pm UTC tomorrow will work for me, I will reach out via email.

**Comment by rsignell-usgs** [comment]
Great!  Did you send it to rsignell@usgs.gov?

**Comment by peterm790** [comment]
Yes, I have sent a second from an alternate email incase the previous did not make it past a spam filter. My preferred email is petermarsh790@gmail.com
